src/ucs/sys/topo/base/topo.c

```diff
@@ -338,7 +338,8 @@ static void ucs_topo_set_distance(const ucs_sys_dev_distance_t *distance_cfg,
 {
     distance->latency   = distance_cfg->latency;
     distance->bandwidth = UCS_CONFIG_DBL_IS_AUTO(distance_cfg->bandwidth) ?
-                          bw_auto : distance_cfg->bandwidth;
+                                  bw_auto :
+                                  distance_cfg->bandwidth
```

Comment:
i'd leave it as before. I.e.
```suggestion
                                  bw_auto : distance_cfg->bandwidth
```